### DISCUSSION TITLE
Satellite data

### DISCUSSION THREAD

**Discussion by dineshbvadhia** [kraina-ai/srai#445] [Q&A]
Hi. I'm new to srai. Is srai applicable to satellite data? Thanks

**Comment by RaczeQ** [comment]
Hello @dineshbvadhia, sorry for the late reply. Currently `srai` operates only on vector-based OSM and GTFS data and for now we don't have any loaders dedicated to raster/satellite data.
It doesn't mean that it won't be applicable at all - the only missing component is a transformation from rasters to numerical values per given region, after that the same models could be applied (hex2vec, geovex etc).
